data/vidhimakani74.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "Vidhi Makani",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Newbie in tech | Learning Web-Development | Learning from Communities & Opensource ",
+  
+  "tags": ["Open Source", "Web Development", "Fronted" ],
```

Non-blocking: minor spelling error here.
```suggestion
  "tags": ["Open Source", "Web Development", "Frontend" ],
```